src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
+//new class MainApp
```

Comment:
Please don't add redundant empty lines to your code. We don't need them after class declaration or before the method signature.